lib/elasticquery/builder.rb

```diff
@@ -0,0 +1,32 @@
+require 'elasticquery/filters/term'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.